tests/test_helperfuncs.py

```diff
@@ -1814,5 +1814,43 @@ def test_no_ones(self):
         self.assertEqual(sim.inspect('o'), 0)
 
 
+class TestBinaryToOneHot(unittest.TestCase):
+    def setUp(self):
+        pyrtl.reset_working_block()
+
+    def test_simple_binary_to_one_hot(self):
+        bit_position = pyrtl.Input(bitwidth=8, name='bit_position')
+        one_hot = pyrtl.Output(bitwidth=16, name='one_hot')
+        one_hot <<= pyrtl.binary_to_one_hot(bit_position)
```

Comment:
We should also check that `binary_to_one_hot` has the correct bitwidth in both these tests (256 in this example)